deploy/clowdapp.yaml

```diff
@@ -0,0 +1,85 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: notifications
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: notifications
+  spec:
+    envName: ${ENV_NAME}
+    deployments:
+    - name: notifications-backend
+      minReplicas: ${{MIN_REPLICAS}}
+      web: true
+      podSpec:
+        image: ${IMAGE}:${IMAGE_TAG}
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /
+            port: 8000
+            scheme: HTTP
+          initialDelaySeconds: 35
+          periodSeconds: 5
+          successThreshold: 1
+          timeoutSeconds: 120
+        readinessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /
+            port: 8000
+            scheme: HTTP
+          initialDelaySeconds: 35
+          periodSeconds: 5
+          successThreshold: 1
+          timeoutSeconds: 120
```

Comment:
Liveness and readiness probe seem wrong they don't match the current ones:
https://github.com/RedHatInsights/e2e-deploy/blob/master/templates/notifications/notifications-backend.yml

Reply:
Fixed